relay/stratoschain/txbuild.go

```diff
@@ -63,6 +63,25 @@ func BuildCreateIndexingNodeMsg(networkID, token, moniker string, pubKey []byte,
 	)
 }
 
+// Stratos-chain 'register' module
```

Comment:
Still register module? what this comment means?

Reply:
yep this indicates it's a tx of Register Module